### ISSUE TITLE
Theme should be accessible in is own menu

### ISSUE THREAD

**Issue by stherm** [KevinJump/Our.Umbraco.BackOfficeThemes#3]
Theme menu should be accessible from it's own menu, so user that doesn't have access to settings tab can change their theme.

**Comment by KevinJump** [comment]
yeah, ideally i would want it inside the user menu (so when you click on your avatar in the top right) - not sure how extensible that is though :(

**Comment by KevinJump** [comment]
v2 now does this - because the user dashboard (behind the avtar top right) is now an option in Umbraco 8.16.x and Umbraco 9. So we've re-written to put the menu there. 

![](https://github.com/KevinJump/Our.Umbraco.BackOfficeThemes/blob/shared/main/screenshots/theme-dashboard.png?raw=true)
